inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -185,7 +196,35 @@ def from_curves_file(cls,
 
         return cls(coils, num_field_periods)
 
-    def simulate(self):
+    def simulate(
+        self,
+        simulator: Simulator = Simsopt(),
+        resource_pool_id: Optional[UUID] = None,
+        run_async: bool = False,
+        magnetic_field_filename: str = 'magnetic_field.npz',
+        plasma_surface_filename: str = 'input.QA',
```

Comment:
I don't think this should have a default.

Reply:
I agree but it there a way to avoid the Pylint error then?

Reply:
Unfortunately the Scenario super class does not have `*args` in the simulate function so I think I will have to leave it with a default name.